app/app.go

```diff
@@ -694,8 +729,6 @@ func (app *OsmosisApp) BeginBlocker(ctx sdk.Context, req abci.RequestBeginBlock)
 
 // EndBlocker application updates every end block.
 func (app *OsmosisApp) EndBlocker(ctx sdk.Context, req abci.RequestEndBlock) abci.ResponseEndBlock {
-	// Process the block and ingest data into various sinks.
-	app.IngestManager.ProcessBlock(ctx)
```

Comment:
Note: we now use SDK streaming API wiring so the original ingest manager logic is not needed anymore